### PR TITLE
Add review page

### PR DESCRIPTION
**Pull request by calvinlu3**
This fixes https://github.com/oncokb/oncokb/issues/3646

### CODE REVIEW

**Comment by zhx828** [comment]
The feature overall looks good. Two minor fixes needed before code review
- after deleting Tx. The change in the review is not expected PTEN/Oncogenic Mutations/Capivasertib + Fulvestrant
Screenshot 2024-02-01 at 11 48 23 AM
- only bold the title when it's not `disableLeftBorder`

**Comment by calvinlu3** [comment]
The Tx level should be fixed now
image

File: src/main/webapp/app/pages/curation/CurationPage.tsx [hunk 1]
```diff
@@ -239,271 +250,333 @@ const CurationPage = (props: ICurationPageProps) => {
...
+      }
+    } else {
+      if (isReviewFinished) {
+        button = (
+          <Button color="primary" onClick={handleReviewButtonClick}>
+            Review Complete
+          </Button>
+        );
+      } else {
+        return undefined;
```
Comment (zhx828) [inline src/main/webapp/app/pages/curation/CurationPage.tsx:288]: Can this really happen?

**Review by zhx828 — APPROVED** [review]
I scanned the PR and didn't see anywhere needs to change right away. We will add a lot of tests for this component, we will revisit again. Thank you for your hard work on this component!

File: src/main/webapp/app/pages/curation/CurationPage.tsx (hunk 1 above)
Comment (calvinlu3) [reply src/main/webapp/app/pages/curation/CurationPage.tsx:288]: `isReviewFinished` is false by default (and only gets set in the review mode) and when a gene doesn't need review, this condition will be hit. We don't want to show a button in this scenario.

**Pull request merged**